assets/src/dashboard/app/views/myStories/header/test/header.js

```diff
@@ -157,14 +160,27 @@ describe('My Stories <Header />', function () {
         />
       </LayoutProvider>
     );
-    expect(getByText('All Stories (19)')).toBeInTheDocument();
-    expect(getByText('Drafts (9)')).toBeInTheDocument();
-    expect(getByText('Published (10)')).toBeInTheDocument();
+
+    const allStoriesButton = getByRole('button', {
+      name: /Filter stories by All Stories/,
+    });
+
+    const draftsButton = getByRole('button', {
+      name: /Filter stories by Drafts/,
+    });
+    const publishedButton = getByRole('button', {
+      name: /Filter stories by Published/,
+    });
+
+    expect(allStoriesButton).toHaveTextContent('All Stories19');
+    expect(draftsButton).toHaveTextContent('Drafts9');
+    expect(publishedButton).toHaveTextContent('Published10');
```

Comment:
wait... there's a `toHaveTextContent()` that filters out DOM tags?? 🤦 :the-more-you-know:

Reply:
oh, but it's just to test an element with. I'm guessing there no prebuilt `getByTextContent` selector. nvmd

Reply:
i think i got this from a PR you did last week 😝